src/Our.Umbraco.Ditto/ComponentModel/TypeConverters/DittoBaseConverter.cs

```diff
@@ -0,0 +1,175 @@
+﻿namespace Our.Umbraco.Ditto
+{
+    using System;
+    using System.Collections.Generic;
+    using System.ComponentModel;
+    using System.Globalization;
+    using System.IO;
+    using System.Xml;
+
+    using global::Umbraco.Core;
+    using global::Umbraco.Core.Models;
+    using global::Umbraco.Web;
+
+    /// <summary>
+    /// The Ditto base converter containing reusable properties and converting methods for <see cref="IPublishedContent"/> instances.
+    /// All other Ditto converters should inherit from this class.
+    /// </summary>
+    public abstract class DittoBaseConverter : TypeConverter
+    {
+        /// <summary>
+        /// Gets the <see cref="UmbracoHelper"/> for querying published content or media.
+        /// </summary>
+        public static UmbracoHelper UmbracoHelper
+        {
+            get
+            {
+                // Pull the item from the cache if possible to reduce the db access overhead caused by 
+                // multiple reflection iterations for the given type taking place in a single request.
+                return (UmbracoHelper)ApplicationContext.Current.ApplicationCache.RequestCache.GetCacheItem(
```

Comment:
Originally I did it as multiple type converters could be running on the same request. I honestly don't know what happens when an UmbracoHelper is created so I wanted to avoid any db driven or heavy going code.